### DISCUSSION TITLE
Add extract frame

### DISCUSSION THREAD

**Discussion by gfolw** [talmolab/sleap#1270] [Ideas]
I like the feature where you can go to a specific frame number, but would it be possible to **extract individual frames or frames from a specific range** in a new window to analyze/label a certain section of the video and then add it back to the project? This would make it easier to label and relabel predicted frames.

**Comment by gitttt-1234** [comment]
Closing this discussion, this is now supported! You can extract clips via **File > Export Training Package...** or use the clip extraction feature added in [PR #2155](https://github.com/talmolab/sleap/pull/2155), which supports exporting clips as video, `.slp`, and `.pkg.slp`.

You can also do this programmatically with [`sleap-io`](https://github.com/talmolab/sleap-io):

```python
import sleap_io as sio

labels = sio.load_slp("your_project.slp")
subset = labels.extract(range(100, 200))
sio.save_file(subset, "subset.pkg.slp", embed="user")
```
Feel free to open a new discussion if you come across any issues!
